variables.tf

```
@@ -28,6 +28,12 @@ variable "chart_namespace" {
   default     = "default"
 }
 
+variable "chart_namespace_created" {
```

```suggestion
variable "chart_namespace_create" {
```

Calling it `_created` sounds weird.